classes/Cart.php

```diff
@@ -1092,9 +1092,11 @@ public static function cacheSomeAttributesLists($ipa_list, $id_lang)
             ORDER BY ag.`position` ASC, a.`position` ASC'
         );
 
+        $colon = Context::getContext()->getTranslator()->trans(':', [], 'Admin.Global');
```

Comment:
```suggestion
        $colon = Context::getContext()->getTranslator()->trans(': ', [], 'Admin.Global');
```

Reply:
@atomiix Wording validated by @Julievrz  :p

Reply:
@Progi1984 I know but with `:` the finale sentence will be `Size:X-M` instead of `Size: X-M`.

Reply:
Ping @Julievrz

Reply:
In that case we should add a space, thank you @atomiix and sorry @Progi1984 I didn't realize this 😅

Reply:
Thanks @atomiix :heart_eyes:  & @Julievrz